src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -98,6 +100,14 @@ internal PreferencesViewModel PreferencesViewModel
             }
         }
 
+        internal GeometryScalingViewModel GeoScalingViewModel
```

Comment:
any chance we can move this to workspace view model?

Reply:
Moved to WorkspaceViewModel, check the next commit:
https://github.com/DynamoDS/Dynamo/pull/13656/commits/789f51c4a3c106156940924585e1ca40abd9842b